llvm/docs/LangRef.rst

```diff
@@ -3143,7 +3143,10 @@ as follows:
     specified, the default index size is equal to the pointer size. All sizes
     are in bits. The address space, ``n``, is optional, and if not specified,
     denotes the default address space 0. The value of ``n`` must be
-    in the range [1,2^24).
+    in the range [1,2^24). The fifth parameter ``<sentinel>`` specifies the
+    sentinel value of the pointer for the corresponding address space. It
+    currently accepts two values: ``0`` for an all-zero value and ``f`` for a
```

Comment:
It would be nice if we could just use a normal number `0`/`-1` but since `-` is the separator that does work and having to spell out the full numeric all-ones value is even more ugly. While I get the rationale for `f` it is a bit odd having 0 be the other one. I'd say either go for letters in both cases `f`(ull)/`z`(ero) or `a`(ll)/`z`ero?

Another option could be `0` and `~0` for all ones?